inc/template-tags.php

```diff
@@ -985,6 +985,11 @@ function responsive_sidebar_classes( $sidebar_id ) {
 	$count        = ( $widget_count > 0 ) ? $widget_count : 'none';
 
 	echo esc_attr( "widget-count-$count" );
+
+	// Add alternate-footbar class.
+	if ( 'alternate-footbar' === $sidebar_id ) {
+		echo esc_attr( ' alternate-footbar ' );
```

Comment:
This is working great!

I just double checked this and it adds an extra space at the end of the classes. I'm not sure if that's okay or not, but just wanted to make sure you were aware.

![Screen Shot 2021-09-22 at 11 24 30 AM](https://user-images.githubusercontent.com/1828613/134373534-0ac976b1-3a53-4889-8755-a28fb1da88d2.png)

Reply:
Extra, trailing, or leading spaces are allowed. Please refer to this [article](https://stackoverflow.com/questions/21439859/html5-compliant-trailing-space-in-class-attribute) that references the specifications for questions.

The trailing space was left as a precautionary measure.

I [reviewed all instances](https://github.com/search?q=org%3Abu-ist+responsive_sidebar_classes&type=code) of where this function is currently used and now feel comfortable removing the trailing space with the next commit.

Please note, this template is overwritten in several themes, and the function is also used in other circumstances for other named footers. The alternate footbar class will only be added when the sidebar ID is `alternate-footbar`.

Additionally, this change only affects responsive 2x. If [3x](https://github.com/bu-ist/responsive-framework-3x/blob/38e9d5fc08c22311ff0ac0bb3ab8a3023539c374/inc/template-tags.php#L874) requires this change please open an issue/bring any existing issues to Sean's attention for prioritizing.